behavior_tests/src/ngt-invalid-json7/do_test.py

```diff
@@ -21,22 +21,27 @@ def setup_test():
 def migrate_test():
     data = []
     ret = []
+    iter = 0
     with open("compile_commands.json", 'r') as f:
         data = f.readlines()
         for line in data:
+            if iter == 1:
+                line = line.replace("directory_placeholder", os.getcwd().replace("\\", "\\\\"))
             if "directory_placeholder" in line:
-                ret.append("        \"output\": \"aaaaa\",\n")
-            line = line.replace("directory_placeholder", os.getcwd().replace("\\", "\\\\"))
+                iter += 1
```

Comment:
Sine 'Output' and 'Arguments' in compile_commands.json could be consumed by SYCLomatic, the test case should add these two fields in compile_commands.json to verify this feature, instead of remove them.

Reply:
In the new PR for this, I have changed the name of these tests to 'standard_compilation_db_json' and I am keeping 'Arguments' and 'Output' in compilation db to verify that dpct consumes these arguments.